src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -285,14 +289,15 @@ private void SetCredentialsOptions()
                 VerboseTrace("Set credentials options");
             }
 
-            private void SetCookieOption()
+            internal void SetCookieOption(Uri uri)
             {
                 if (!_handler._useCookie)
                 {
+                    SetCurlOption(CURLoption.CURLOPT_COOKIE, IntPtr.Zero);
```

Comment:
What cookies is this overriding by passing IntPtr.Zero?  Is it that libcurl will use cookies its previously stored, even though neither COOKIEFILE nor COOKIEJAR are being set?